src/main/java/org/humanistika/oxygen/tei/completer/TeiCompleter.java

```diff
@@ -235,7 +235,7 @@ private AuthenticationType asClientFactoryAuthenticationType(@Nullable final Aut
         }
     }
 
-    protected class AutoCompleteXPaths {
+    public class AutoCompleteXPaths {
```

Comment:
Why is this visibility change needed?

Reply:
The GUI source files are not in the same package as `TeiCompleter` packages
```
package org.humanistika.oxygen.tei.completer.GUI;
package org.humanistika.oxygen.tei.completer;
```
Even trying to set it package public doesn't not work in this case

Reply:
Okay cool, I see that now, thanks :-)